src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -189,9 +189,7 @@ namespace winrt::TerminalApp::implementation
     }
 
     AppLogic::AppLogic() :
-        _dialogLock{},
-        _loadedInitialSettings{ false },
-        _settingsLoadedResult{ S_OK }
+        _reloadState{ std::chrono::milliseconds(100), []() { ApplicationState::SharedInstance().Reload(); } }

```

Comment:
Similar to the existing settings-reload throttler, this does the same but for state.json.
Since `til::throttled_func` cannot be copied or moved, this type must be initialized in the member list initializer (instead of more ergonomically in the constructor function body).